app/src/main/java/org/jellyfin/androidtv/ui/home/HomeFragment.kt

```diff
@@ -106,6 +109,25 @@ class HomeFragment : StdBrowseFragment(), AudioEventListener {
 			// Update the views before creating rows
 			views = callApi<ItemsResult> { apiClient.GetUserViews(TvApp.getApplication().currentUser!!.id, it) }
 
+			// This is kind of ugly, but it mirrors how web handles the live TV rows on the home screen
+			// If we can retrieve one live TV recommendation, then we should display the rows
+			callApi<ItemsResult> {
```

Comment:
Right now it will always make this additional API call which slows down home section loading. The live tv access might be disabled already so we'd better check that first. You could use `TvApp.getApplication().currentUser!!.policy.enableLiveTvAccess` here instead of in the `addSection` function to set `includeLiveTvRows` to false before running the call.

Reply:
Good call! I've updated it now.